app/views/publishers/ats_api/v1/vacancies/_show.json.jbuilder

```diff
@@ -1,23 +1,23 @@
+json.id vacancy.id
 json.external_advert_url vacancy.external_advert_url
 json.publish_on vacancy.publish_on
 json.expires_at vacancy.expires_at
 json.job_title vacancy.job_title
-json.skills_and_experience vacancy.skills_and_experience
+json.job_advert vacancy.job_advert
 json.salary vacancy.salary
 json.benefits_details vacancy.benefits_details
-json.starts_on vacancy.starts_on if vacancy.starts_on.present?
+json.starts_on vacancy.start_date_type == "specific_date" ? vacancy.starts_on : vacancy.other_start_date_details
 json.visa_sponsorship_available vacancy.visa_sponsorship_available
-json.is_job_share vacancy.is_job_share if vacancy.is_job_share.present?
+json.is_job_share vacancy.is_job_share
 json.external_reference vacancy.external_reference
 json.job_roles vacancy.job_roles
 json.schools do
-  json.school_urns(vacancy.organisations.map(&:urn))
+  json.school_urns(vacancy.organisations.filter_map(&:urn))
+  json.trust_uid(vacancy.trust_uid)
```

Comment:
defined this in the vacancy so we can obtain "what trust the vacancy belongs to", either if the vacancy belongs to the school group itself or to one/multiple schools for that trust.